tests/TestCase.php

```diff
@@ -72,4 +76,18 @@ protected function useRealUuid()
             return Ray::create($this->client);
         });
     }
+
+    protected function requireMinimumLaravelVersion($version)
```

Comment:
Take a look at this neat PHP function: https://www.php.net/manual/en/function.version-compare.php

Let's make us of that.

Reply:
Nice! Thanks for that 🔥